src/com/epam/andrei_eremenko/task_01/module/optimal_stopping/RatesCalculator.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.andrei_eremenko.task_01.module.optimal_stopping;
+
+
+class RatesCalculator {
+
+    static void sumRates(int[] grooms) {
+        int flag = 0;
+        int nextStep = OptimalStoppingAlgorithm.STEEP;
+
+        while (nextStep <= OptimalStoppingAlgorithm.QUANTITY_OF_GROOMS) {
+
+            for (int i = nextStep - 1; i < grooms.length; i++) {
+                int temp = grooms[nextStep - 1];
+
+                if (grooms[i] > temp) {
+                    OptimalStoppingAlgorithm.ratesAmount[flag] += grooms[i];
+                    flag++;
+                    break;
+                } else if (i == grooms.length - 1) {
```

Comment:
I believe there is mistake. You have two exactly the same branches in `if else` condition.